code/renderers/svelte/templates/PreviewRender.svelte

```diff
@@ -15,37 +15,27 @@
     props = {},
     /** @type {{[string]: () => {}}} Attach svelte event handlers */
     on,
-    Wrapper,
-    WrapperData = {},
   } = storyFn();
 
   // reactive, re-render on storyFn change
-  $: ({ Component, props = {}, on, Wrapper, WrapperData = {} } = storyFn());
```

Comment:
I believe these `Wrapper` and `WrapperData` was causing the issue at #19671 , because they're never set, in `decorators.ts` they were called `decorator` and `decoratorProps`